packages/web3-utils/test/fixtures/converters.ts

```diff
@@ -300,9 +300,21 @@ export const fromWeiValidData: [[Numbers, EtherUnits], string][] = [
     [[1.9999999999999991611392e+22, 'ether'], '19999.999999999991611392'],
 ];
 
-export const toWeiValidData: [[Numbers, EtherUnits], string][] = [
+export const toWeiValidData: [[Numbers, EtherUnits], Numbers][] = [
 	...conversionBaseData,
 	[['255', 'wei'], '0xFF'],
+	[['255', 'wei'], '0xFF'],
```

Comment:
This line is doublicate. Was it meant to have differnt values? Or, should this be deleted?